home/src/main/resources/rdf/tbox/filegraph/vitro-dynamic-api.owl

```diff
@@ -472,13 +488,14 @@
     <!-- https://vivoweb.org/ontology/vitro-dynamic-api#operation -->
 
     <owl:Class rdf:about="https://vivoweb.org/ontology/vitro-dynamic-api#operation">
-        <rdfs:label xml:lang="en-US">Operation</rdfs:label>
+        <rdfs:label xml:lang="en-US">Operation interface</rdfs:label>
     </owl:Class>
 
 
     <!-- https://vivoweb.org/ontology/vitro-dynamic-api#template -->
 
     <owl:Class rdf:about="https://vivoweb.org/ontology/vitro-dynamic-api#template">
+        <rdfs:subClassOf rdf:resource="https://vivoweb.org/ontology/vitro-dynamic-api#parameterReceiver"/>
```

Comment:
In your example (https://app.diagrams.net/#G176In5cQ-tWecx0aP0ILBTODuCoiYk-0o) N3Template is parameter receiver and provider?

Reply:
My example has a mistake. It shouldn't be a provider